src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -257,12 +257,20 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
         )
 
         if (!hasLaunchJson) {
-            vscode.window.showErrorMessage(
-                localize(
-                    'AWS.sam.debugger.noLaunchJson',
-                    'AWS SAM: To debug a Lambda locally, first create a launch.json from the VS Code "Run" panel'
+            vscode.window
+                .showErrorMessage(
+                    localize(
+                        'AWS.sam.debugger.noLaunchJson',
+                        'AWS SAM: To debug a Lambda locally, create a launch.json from the Run panel, then select a configuration.'
+                    ),
+                    localize('AWS.gotoRunPanel', 'Run panel')
```

Comment:
Consider the text "Go to Run panel"

Reply:
I intentionally avoided that, since "go to" is implied by it being a button.